sdks/python/container/boot.go

```diff
@@ -59,6 +60,18 @@ const (
 
 func main() {
 	flag.Parse()
+
+	if *workerPool == true {
+		args := []string{
+			"-m",
+			"apache_beam.runners.worker.worker_pool_main",
+			"--service_port=50000",
+			"--container_executable=/opt/apache/beam/boot",
+		}
+		log.Printf("Starting Python SDK worker pool: python %v", strings.Join(args, " "))
+		log.Fatalf("Python SDK worker pool exited: %v", execx.Execute("python", args...))
```

Comment:
Should we start this after artifact staging? Otherwise each of these processes will have to set their own environment. In addition to multiple downloads, they also risk running installation in parallel and failing.

Reply:
I'm working on the locking that prevents multiple processes from downloading artifacts and running installation concurrently, since this needs to be done only once. I would prefer to defer this to a follow-up PR though.

Reply:
Sounds good. Do you mind adding a TODO here for that follow up PR.